keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -328,10 +329,20 @@ open class SqlResourceRepository(
                 .execute()
             }
           }
-          .map { (_, kind, metadata, spec) ->
+      }
+        .map { (uid, kind, metadata, spec) ->
+          try {
             constructResource(kind, metadata, spec)
+          } catch (e: Exception) {
```

Comment:
This logic will ignore updates regardless of the exception that occurs. I worry about situations where constructResource throws an exception for a transient issue (e.g., networking issue connecting with the database). Couple of thoughts around that:

* We need to feed back info to a human being when a resource gets ignored. It should hopefully be rare enough that we can handle these as they come in. Since we're logging this as an error when it happens, I think we're good here, because we'll get alerted by RADAR.
* Human operator needs to know how to fix things when a resource gets into a bad state due to a transient issue. Looks like the operation is simple enough (just set ignore to `false` in the database record), but we need to make sure everyone knows how to do this.

Reply:
(Note that I saw we're logging as an error when the exception gets caught upstream, so no need to add any additional log statements).